vendor/bat-native-ledger/src/bat/ledger/internal/common/bind_util.cc

```diff
@@ -114,6 +114,7 @@ std::string FromPromotionToString(const ledger::PromotionPtr info) {
   promotion.SetIntKey("suggestions", info->suggestions);
   promotion.SetBoolKey("status", static_cast<int>(info->status));
   promotion.SetKey("credentials", std::move(credentials));
+  promotion.SetBoolKey("legacyClaimed", static_cast<int>(info->legacyClaimed));
```

Comment:
not sure why cast is needed variable is already bool

Reply:
Just following the example with 'status' - I'm not used to mojom requirements yet. I'll remove it :+1:

Reply:
Done